registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/utils/Utilities.java

```diff
@@ -703,35 +612,7 @@ public JSONObject idrepoRetrieveIdentityByRid(String regId) throws ApisResourceA
 		return null;
 	}
 
-	/**
-	 * Calculate age.
-	 *
-	 * @param applicantDob the applicant dob
-	 * @return the int
-	 */
-	private int calculateAge(String applicantDob) {
-		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(), "",
-				"Utilities::calculateAge():: entry");
-
-		DateFormat sdf = new SimpleDateFormat(dobFormat);
-		Date birthDate = null;
-		try {
-			birthDate = sdf.parse(applicantDob);
-
-		} catch (ParseException e) {
-			regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
-					"", "Utilities::calculateAge():: error with error message "
-							+ PlatformErrorMessages.RPR_SYS_PARSING_DATE_EXCEPTION.getMessage());
-			throw new ParsingException(PlatformErrorMessages.RPR_SYS_PARSING_DATE_EXCEPTION.getCode(), e);
-		}
-		LocalDate ld = new java.sql.Date(birthDate.getTime()).toLocalDate();
-		Period p = Period.between(ld, LocalDate.now());
-		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(), "",
-				"Utilities::calculateAge():: exit");
 
-		return p.getYears();

```

Comment:
These changes are again related to any bugfix ?

Reply:
why are these methods moved to the new file ?

Reply:
These changes are related to circular dependency which I got in registration processor after migration while running application.So as suggested by Loga I have break the dependency  and moved code to utility